voice-client-javascript/tests/admin/login.test.js

```diff
@@ -0,0 +1,59 @@
+const helpers = require("../../../test/test-helper");
+
+// Defined after Runtime is available
+let loginFunction;
+let createToken;
+
+describe("voice-client-javascript/admin/login", () => {
+  beforeAll(() => {
+    process.env.ADMIN_PASSWORD = "supersekret";
+    const runtime = new helpers.MockRuntime();
```

Comment:
@dkundel @philnash Used here. It feels good and keeps `path` behind the scenes. Added benefit of explicit error when not used.